build.sbt

```diff
@@ -21,6 +24,21 @@ libraryDependencies ++= Seq(
 routesGenerator := InjectedRoutesGenerator
 
 
+// Front-end assets config
+val bundle = taskKey[Pipeline.Stage]("JSPM bundle")
+
+bundle := { mappings =>
+  val log = streams.value.log
+  log.info("Running JSPM bundle")
+  val cmd = Process("npm run bundlejs", baseDirectory.value) !< log
+  if (cmd != 0) sys.error(s"Non-zero error code for `npm run bundlejs`: $cmd")
+  mappings
+}
+
+pipelineStages := Seq(bundle, digest, gzip)
```

Comment:
Hey @jamespamplin I am interested to know if this works with incremental changes?